<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IHMDanger</class>
 <widget class="QMainWindow" name="IHMDanger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>348</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>812</width>
    <height>348</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IHMDanger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_frame_com">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Communication µc :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_com_status">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Status :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_com_status_output">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Non Connecté</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_com_connexion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Connexion</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_com_deconnexion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Deconnexion</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_com_portcom">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>42</width>
       <height>25</height>
      </rect>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_com_portcom">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Port COM</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_frame_signal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Signalisation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_signal_led1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Led1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_signal_led2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Led2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_signal_afficher">
     <property name="geometry">
      <rect>
       <x>132</x>
       <y>80</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Afficher</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_signal_eteindre">
     <property name="geometry">
      <rect>
       <x>132</x>
       <y>110</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Eteindre</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_signal_nb_chiffre_afficher">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Chiffre à afficher :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_signal_nb_chiffre_afficher">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>100</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_wd_donnee_lues">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Donnée lues</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_wd_trame_extraite">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Trame extraite</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_wd_trame_extraite">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_wd_trame_stockee">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Trame stockée</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_wd_nb_octets_disp">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nb d'octets disponibles</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_wd_nb_octets_trame_extr">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nb octets trame extraite</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_wd_nb_octets_trame_stock">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nb octets trame stockée</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_wd_nb_octets_lus">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>180</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nb d'octets lus</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_wd_nb_octets_disp">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>180</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_wd_nb_octets_lus">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>180</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_wd_nb_octets_trame_extr">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>260</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_wd_nb_octets_trame_stock">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>310</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_wd_donnee_lues">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>441</width>
      <height>131</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_wd_affichage_ameliore">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Affichage +</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
